eng/tox/environment_exclusion_list.py

```diff
@@ -1,382 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Seeing as this moved to ci_tools, would you mind updating the link in this dev doc: https://github.com/Azure/azure-sdk-for-python/blob/711d92986d8278c7e24048d98d3bcf5951eb5764/doc/dev/static_type_checking.md?plain=1#L301

Reply:
Absolutely,  thanks for the heads up!

Reply:
Done!